docs/reference/modules/task.md

```diff
@@ -0,0 +1,97 @@
+# `@apostrophecms/task`
+
+**Alias:** `apos.task`
+
+This module allows support modules in creating command line tasks. It also provides utilities for generating request objects when one is not available and is needed.
+
+Command line tasks are invoked with the general structure:
+
+```bash
+node app module-name:task-name --arguments
+
+# Example
+node app @apostrophecms/migration:migrate
+node app article:generate --total=20
+```
+
+Apostrophe is fully initialized before a task is run, except that it does not listen for connections. We may access all general Apostrophe features in a task.
+
+## Related documentation
+
+- [Module task configuration](/reference/module-api/module-overview.md#tasks-self)
+
+## Featured methods
+
+The following locales belong to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/task/index.js) for all methods that belong to this module.
+<!-- Some are used within the module and would just create noise here. -->
+
+::: danger
+`getReq` and the related methods below should be used very carefully. It is easy to accidentally grant admin-level (or other higher role) access when lower level permissions are better applied. Those methods are primarily meant for writing unit tests and CLI tasks.
+
+It is almost always the case that an existing request object should be used instead when available (e.g., in an API route handler or event handler such as `beforeInsert`).
+:::
+
+### `invoke(name, args, options)`
+
+For use when you wish to execute an Apostrophe command line task from server-side code and continue with other work. This avoids using the command line directly or using the Node.js `child_process` module. The method runs asynchronously (returns a Promise).
```

Comment:
And then continue with other work.

I would add "and can be awaited." I try to orient my language more toward async/await syntax because it is more accessible.

Reply:
I'm not totally sure "awaited" is well adopted as an verb for this. I think we do need to phrase it differently, but we can refer to that syntax.